packages/backend/src/config/env.ts

```diff
@@ -1,35 +1,39 @@
-import { z } from 'zod'
-
-// import * as dotenv from 'dotenv'
-// dotenv.config({})
-
-const envSchema = z.object({
-  PORT: z.coerce.number(),
-  NODE_ENV: z.string().default('development'),
-  DB_URL: z.string(),
-  JWT_ACCESS_TOKEN_SECRET: z.string(),
-  JWT_ACCESS_TOKEN_EXPIRATION_TIME: z.coerce.number(),
-  JWT_REFRESH_TOKEN_SECRET: z.string(),
-  JWT_REFRESH_TOKEN_EXPIRATION_TIME: z.coerce.number(),
-  RAPYD_API: z.string(),
-  RAPYD_ACCESS_KEY: z.string(),
-  RAPYD_SECRET_KEY: z.string()
-})
-
-type ENV = z.infer<typeof envSchema>
-
-const env: ENV = (() => {
-  const result = envSchema.safeParse(process.env)
-
-  if (!result.success) {
-    console.error(
-      'Error parsing environment variables:',
-      result.error.flatten()
-    )
-    process.exit(1)
-  }
-
-  return result.data
-})()
+// import { z } from 'zod'
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const d = require('dotenv')
+
+// const envSchema = z.object({
+//   PORT: z.coerce.number(),
+//   NODE_ENV: z.string().default('development'),
+//   DB_URL: z.string(),
+//   JWT_ACCESS_TOKEN_SECRET: z.string(),
+//   JWT_ACCESS_TOKEN_EXPIRATION_TIME: z.coerce.string(),
+//   JWT_REFRESH_TOKEN_SECRET: z.string(),
+//   JWT_REFRESH_TOKEN_EXPIRATION_TIME: z.coerce.number(),
+//   RAPYD_API: z.string(),
+//   RAPYD_ACCESS_KEY: z.string(),
+//   RAPYD_SECRET_KEY: z.string()
+// })
```

Comment:
Instead of commenting out the `envSchema` we can specify defaults for our environment variables:
```
PORT: z.coerce.number().default(3000)
```

Reply:
indeed

Reply:
can we set some test values on jest setup?

Reply:
well, i guess the test values will actually be the default ones, since we expect to have them in the docker environment when we want to demo